src/components/Heading/Heading.tsx

```diff
@@ -49,28 +52,36 @@ type Props = {
    * HTMLのfor属性。label要素の場合に使用
    */
   htmlFor?: string;
+  /**
+   * 太字とするかどうか、falseの場合はnormal
+   * @default true
+   */
+  bold?: boolean;
 };
 
 const Heading: FC<PropsWithChildren<Props>> = ({
-  textAlign = 'left',
+  textAlign,
   children,
   primaryIcon,
   accentIcon,
   whiteIcon,
   size = 'md',
-  variant = 'secondary',
+  color = 'main',
   leadingBorder,
   as: HeadingComponent = 'p',
   id,
   htmlFor,
+  bold = true,
```

Comment:
Instead of `bold`, `normal` prop orz `weight` prop are also possible. But unified with `<Text>`.